AI/DB_interface.py

```
@@ -0,0 +1,54 @@
+        close(conn) # closes the connection
+        return False
+    
+def get_data_colums(query, params=()): 
+    conn = connect() # connects to the DB
+    try:
+        cursor = conn.cursor() # creates a cursor object
+        cursor.execute(query, params) # executes the query with params
+        rows = cursor.fetchall() # fetches all the data
+        columns = [desc[0] for desc in cursor.description] # gets the column names
+        close(conn) # closes the connection
+        return rows, columns # returns the data and column names
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)